### ISSUE TITLE
WindowsCentralProcessor#getInterrupts return wrong value

### ISSUE THREAD

**Issue by retomerz** [oshi/oshi#602]
Hello,

I have just verified that WindowsCentralProcessor#getInterrupts returns a wrong value.

Steps do reproduce:

1. Open Windows Peformance Monitor and add this counter from the local machine:
Object: Processor
Instance: _Total
Counter: Interrupts/s
Set scale to 1.0

2. Start Process Explorer (SysInternal)
Open System Information -> CPU
The dialog shows a value "Interrupt Delta" in the CPU section.

3. Start following OSHI demo program:

```
import oshi.SystemInfo;
import oshi.hardware.CentralProcessor;
import java.sql.Timestamp;

class Test {
  public static void main(String[] args) throws Exception {
    CentralProcessor cp = new SystemInfo().getHardware().getProcessor();
    while (true) {
      System.out.println(new Timestamp(System.currentTimeMillis()) + ": " + cp.getInterrupts());
      Thread.sleep(1000);
    }
  }
}
```

Compare the values of these 3 sources:
1 und 2 perfectly matches. The output of 3 (OSHI) does not match.
Even when you caculate the delta between current and previous value between the 1 second sleep,
the value is wrong. 

Tested with Windows 2012 R2 and OSHI version 3.8.0
> C:\Windows\System32>ver
> Microsoft Windows [Version 6.3.9600]

Here is the output of my system which performs 6'000-12'000 interrupts/s.
The OSHI output below shows that, according to OSHI, my system performs only 1-2 interrupts/s.

Performance Monitor and Process Explorer:
![image](https://user-images.githubusercontent.com/9657405/44850137-96b1bf00-ac5c-11e8-929b-540076cb3cee.png)

OSHI Output:
```
2018-08-30 13:47:00.481: 19998
2018-08-30 13:47:01.481: 19998
2018-08-30 13:47:02.481: 19999
2018-08-30 13:47:03.481: 20000
2018-08-30 13:47:04.481: 20000
2018-08-30 13:47:05.481: 20001
2018-08-30 13:47:06.481: 20002
2018-08-30 13:47:07.481: 20003
2018-08-30 13:47:08.481: 20003
2018-08-30 13:47:09.481: 20004
2018-08-30 13:47:10.482: 20005
2018-08-30 13:47:11.482: 20005
2018-08-30 13:47:12.482: 20006
2018-08-30 13:47:13.497: 20007
2018-08-30 13:47:14.508: 20007
2018-08-30 13:47:15.508: 20008
2018-08-30 13:47:16.509: 20009
2018-08-30 13:47:17.509: 20009
2018-08-30 13:47:18.509: 20010
2018-08-30 13:47:19.509: 20011
2018-08-30 13:47:20.524: 20011
2018-08-30 13:47:21.524: 20012
2018-08-30 13:47:22.525: 20013
2018-08-30 13:47:23.525: 20013
2018-08-30 13:47:24.525: 20014
2018-08-30 13:47:25.525: 20015
2018-08-30 13:47:26.525: 20015
2018-08-30 13:47:27.525: 20016
2018-08-30 13:47:28.525: 20017
2018-08-30 13:47:29.525: 20017
2018-08-30 13:47:30.529: 20018
2018-08-30 13:47:31.531: 20019
2018-08-30 13:47:32.534: 20019
2018-08-30 13:47:33.534: 20020
2018-08-30 13:47:34.534: 20021
2018-08-30 13:47:35.534: 20021
2018-08-30 13:47:36.545: 20022
2018-08-30 13:47:37.545: 20023
2018-08-30 13:47:38.545: 20023
2018-08-30 13:47:39.545: 20024
2018-08-30 13:47:40.545: 20025
2018-08-30 13:47:41.545: 20025
2018-08-30 13:47:42.545: 20026
2018-08-30 13:47:43.545: 20027
2018-08-30 13:47:44.555: 20027
2018-08-30 13:47:45.555: 20028
2018-08-30 13:47:46.555: 20029
2018-08-30 13:47:47.555: 20029
2018-08-30 13:47:48.555: 20030
2018-08-30 13:47:49.559: 20031
2018-08-30 13:47:50.561: 20031
2018-08-30 13:47:51.561: 20032
2018-08-30 13:47:52.561: 20033
2018-08-30 13:47:53.561: 20033
2018-08-30 13:47:54.561: 20034
2018-08-30 13:47:55.562: 20034
2018-08-30 13:47:56.562: 20035
2018-08-30 13:47:57.562: 20036
2018-08-30 13:47:58.568: 20037
2018-08-30 13:47:59.568: 20037
2018-08-30 13:48:00.578: 20038
2018-08-30 13:48:01.579: 20039
2018-08-30 13:48:02.579: 20039
2018-08-30 13:48:03.579: 20040
2018-08-30 13:48:04.579: 20041
2018-08-30 13:48:05.581: 20041
2018-08-30 13:48:06.592: 20042
2018-08-30 13:48:07.592: 20043
2018-08-30 13:48:08.593: 20044
2018-08-30 13:48:09.593: 20044
2018-08-30 13:48:10.596: 20045
2018-08-30 13:48:11.596: 20046
2018-08-30 13:48:12.596: 20046
2018-08-30 13:48:13.596: 20047
2018-08-30 13:48:14.603: 20048
2018-08-30 13:48:15.603: 20049
2018-08-30 13:48:16.603: 20049
2018-08-30 13:48:17.603: 20050
2018-08-30 13:48:18.603: 20051
2018-08-30 13:48:19.603: 20051
2018-08-30 13:48:20.603: 20052
2018-08-30 13:48:21.606: 20052
2018-08-30 13:48:22.606: 20053
2018-08-30 13:48:23.607: 20054
2018-08-30 13:48:24.607: 20054
2018-08-30 13:48:25.607: 20055
2018-08-30 13:48:26.608: 20056
2018-08-30 13:48:27.608: 20056
2018-08-30 13:48:28.608: 20057
2018-08-30 13:48:29.608: 20058
2018-08-30 13:48:30.611: 20058
2018-08-30 13:48:31.611: 20059
2018-08-30 13:48:32.613: 20060
2018-08-30 13:48:33.613: 20060
2018-08-30 13:48:34.613: 20061
2018-08-30 13:48:35.629: 20062
2018-08-30 13:48:36.639: 20062
2018-08-30 13:48:37.639: 20063
2018-08-30 13:48:38.639: 20063
2018-08-30 13:48:39.639: 20064
2018-08-30 13:48:40.641: 20065
2018-08-30 13:48:41.641: 20066
2018-08-30 13:48:42.641: 20066
2018-08-30 13:48:43.641: 20067
2018-08-30 13:48:44.649: 20068
2018-08-30 13:48:45.649: 20068
2018-08-30 13:48:46.649: 20069
2018-08-30 13:48:47.649: 20070
2018-08-30 13:48:48.651: 20070
2018-08-30 13:48:49.651: 20071
2018-08-30 13:48:50.651: 20071
2018-08-30 13:48:51.651: 20072
2018-08-30 13:48:52.651: 20073
2018-08-30 13:48:53.651: 20073
2018-08-30 13:48:54.651: 20074
2018-08-30 13:48:55.651: 20075
2018-08-30 13:48:56.652: 20076
2018-08-30 13:48:57.652: 20076
2018-08-30 13:48:58.652: 20077
2018-08-30 13:48:59.668: 20078
```

**Comment by dbwiddis** [comment]
The output you are seeing from PerfMon is in 100-ns ticks.  OSHI's output is in millisecond ticks.

**Comment by dbwiddis** [comment]
Or... no. This is a count of interrupts, not time. Perhaps the division by 10,000 is wrong.  I'll look into this.

**Comment by dbwiddis** [comment]
Yeah, looks like a copypaste from the other tick counters (where a 100-ns to ms conversion was needed) slipped through.  Line 452 of `WindowsCentralProcessor.java` should not have the division by 10K.

Feel free to submit a PR if you want it fixed in the next few hours; otherwise I'll have it done by tomorrow.

**Comment by retomerz** [comment]
@dbwiddis Maybe I misunderstood you last comment, but as far as I see the division is still performed (and I can not see that this was removed with #606 with commit 552151458b068e04cc322ea3082be062f881cb9e):
https://github.com/oshi/oshi/blob/master/oshi-core/src/main/java/oshi/hardware/platform/windows/WindowsCentralProcessor.java#L436

**Comment by dbwiddis** [comment]
Oops.

I did make the change, honest!   I also had a lot of other changes that went badly so I undid all my changes and put back the ones that worked.   But forgot to put this one back.

**Comment by dbwiddis** [comment]
Really fixed in #608!
